src/Raven.Client/Constants.cs

```diff
@@ -103,6 +103,15 @@ private Configuration()
             public const string ClientId = "Configuration/Client";
         }
 
+        public class Counters
```

Comment:
Use static class here, instead.

Reply:
This is ok actually.

Reply:
Why, this is using a private ctor to avoid instantiation. A static class is clearer in intent.
This is there as a namespace scoping mechanism only, after all.

Reply:
done